### DISCUSSION TITLE
Unexpected Server Respons 400

### DISCUSSION THREAD

**Discussion by ambimuhammad** [kirimwa-id/forum#116] [Tanya Jawab] [answered]
Salam Mas Rio, 
mau tanya, kenapa ketika mau reconnect selalu muncul seperti ini ya? 
pas reconnect berhasil, pas dicek muncul seeprti ini.. device juga masih ngelink

terima kasih

![IMG_20220422_223258](https://user-images.githubusercontent.com/11288607/164747365-17378e48-6612-48bb-9e73-ed0cb67439a1.jpg)

**Comment by rioastamal** [comment, marked as answer]
@ambimuhammad Terima kasih laporannya. Dengan status disconnect tersebut apa masih bisa mengirim pesan?

**Reply by ambimuhammad** [reply]
Sekarang sudah bisa Mas, maaf ternyata ada kesalahan di token tadi
